importScripts/sampleRecords/library.destination.json

```diff
@@ -113,9 +262,90 @@
       }
     },
     {
-      "object_id":"4",
-      "id":"4",
-      "idno":"1133",
+      "object_id":"8",
+      "id":"8",
```

Comment:
This record had accession no extracted but the extract was a duplicate, so it is replaced with nothing.